cpp/log/log_lookup-inl.h

```diff
@@ -68,8 +69,7 @@ void LogLookup<Logged>::UpdateFromSTH(const ct::SignedTreeHead& sth) {
     // handled more gracefully. E.g. we could retry a failed update
     // a number of times -- but until we know under which conditions
     // the database might fail (database busy?), just die.
-    CHECK_EQ(Database<Logged>::LOOKUP_OK,
-             db_->LookupByIndex(sequence_number, &logged))
+    CHECK(it->GetNextEntry(&logged))
```

Comment:
Yeah, I agree, if we have an STH in the database, but not the entries for it, something is seriously wrong, so blowing up is okay.